src/cli/cli_mac_filter.cpp

```diff
@@ -288,6 +288,25 @@ template <> otError MacFilter::Process<Cmd("addr")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli macfilter rss
```

Comment:
Aren't these rss ones from the other PR?  https://github.com/openthread/openthread/pull/9442

Reply:
good catch, thx! Resolved as we discussed.

Reply:
good catch, thanks.  Fixed now as discussed.